src/app/actions/equipment/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './a_equipment_create';
 export * from './a_equipment_delete';
 export * from './a_equipment_reservation_create';
 export * from './a_equipment_reservation_delete';
+export * from './a_quantity_add';
```

Comment:
Wrong file name. Should be :
```suggestion
export * from './a_equipment_quantity_add';
```